docs/ref/contrib/gis/install/index.txt

```diff
@@ -59,7 +59,7 @@ supported versions, and any notes for each of the supported database backends:
 Database            Library Requirements            Supported Versions  Notes
 ==================  ==============================  ==================  =========================================
 PostgreSQL          GEOS, GDAL, PROJ.4, PostGIS     9.3+                Requires PostGIS.
-MySQL               GEOS, GDAL                      5.5+                Not OGC-compliant; :ref:`limited functionality <mysql-spatial-limitations>`.
+MySQL               GEOS, GDAL                      5.5.3+              Not OGC-compliant; :ref:`limited functionality <mysql-spatial-limitations>`.
```

Comment:
this isn't really needed, people don't practically use the early releases as they aren't GA, and never get distributed as OS packages etc.